portfolio/src/main/webapp/script.js

```diff
@@ -19,21 +19,50 @@ const RIGHT_ANSWERS = ["choice-2", "choice-2", "choice-1"];
 const MOVIES =
       ['Austin Powers', 'Ace Ventura', 'Talladega Nights', 'The dictator'];
 
-const QUESTION_2 = {
-    question: "Which soccer team do I support?",
-    choice1: "Manchester City",
-    choice2: "Liverpool"
-}
+const QUIZ = document.getElementById("quiz");
+
+/**
+ *  A class to organize questions for the quiz section
+ */
+class Question{
+
+    /**
+     * @param {string} prompt The question being asked
+     * @param {string} choice1 The first choice
+     * @param {string} choice2 The second choice
+     */
+    constructor(prompt, choice1, choice2){
+        this.prompt_ = prompt;
+        this.choice1_ = choice1;
+        this.choice2_ = choice2;
+    }
+
+    /** @return {string} */
+    getPrompt(){
+        return this.prompt_;
+    }
+
+    /** @return {string} */
+    getChoice1(){
+        return this.choice1_;
+    }
 
-const QUESTION_3 = {
-    question: "Which of the following apps don't I have on my phone?",
-    choice1: "Tiktok",
-    choice2: "Reddit"
+    /** @return {string} */
+    getChoice2(){
+        return this.choice2_;
+    }
 }
 
-const QUESTIONS_LIST = [QUESTION_2,QUESTION_3];
+const question2 = new Question("Which soccer team do I support?", 
+                                "Manchester City", 
+                                "Liverpool");
+
+const question3 = new Question("Which of the following apps don't I have on my phone?", 
+                                "Tiktok", 
+                                "Reddit");
+
+let questionList = [question2,question3];
```

Comment:
I am not changing questionList after this so I want to make it a const but the capitalized const can't sit at the top of the file because the objects in the array are declared later. What do you suggest here?

Reply:
Also, there's the issue of possibly adding more questions to this list later on to make the quiz longer, so do I declare it non-const now knowing that even though I'm not changing it now, I probably will in the future?

Reply:
Yep so 2 things here
 - You can still declare the array with `const questionList = ` if you are adding things to the array later.  
Code for example like:
```
const arr = [];
arr.push('val1');
arr.push('val2');
arr.splice(1);
arr[0] = 'val3';
```
if fine. using `const` would only become an issue if you actually reassign the variable. So for example code like:
```
const arr = ['val1', 'val2'];
arr = ['val3', 'val4];
```
would error out.

As for the naming, since its not deeply immutable you can use camalCase

Reply:
Interesting behavior actually. It's probably b/c the arr variable is just a pointer to the first element and thus anything that wants to change where it's pointing to is an issue but adding stuff afterwards is fine.

Reply:
Yep exactly!